ci-operator/step-registry/code-ready/snc/e2e/test/code-ready-snc-e2e-test-commands.sh

```
@@ -17,8 +17,13 @@ mkdir -p "${HOME}"/.ssh
 mock-nss.sh
 
 # gcloud compute will use this key rather than create a new one
-cp "${CLUSTER_PROFILE_DIR}"/ssh-privatekey "${HOME}"/.ssh/google_compute_engine
-chmod 0600 "${HOME}"/.ssh/google_compute_engine
+PRIVATE_KEY_PATH="${HOME}"/.ssh/google_compute_engine
+cp "${CLUSTER_PROFILE_DIR}"/ssh-privatekey ${PRIVATE_KEY_PATH}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)